src/lsp/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mz-lsp"
```

Comment:
Naming nit: "LSP" is the protocol, so I think we should call this one of: "mz-lsp-server", "mz-language-server", or "mz-lang-server".